lib/morse_translator.rb

```diff
@@ -40,4 +40,22 @@ def initialize
                     " " => " "}
   end
 
+  def translate(english)
+    english.downcase.split("").map { |letter| @dictionary[letter]}.join
+  end
+
+  def morse_to_eng(morse_message)
+    english_array = morse_message.split("  ").map do |morse_word|
+      morse_word_to_eng(morse_word)
+    end
+    english_array.flatten.join.strip
+  end
+
+  def morse_word_to_eng(morse_word)
+    english_word = morse_word.split.map do |morse_character|
+      @dictionary.find{|key, value| value == morse_character}[0]
```

Comment:
This should not be single line syntax. You should break this part out into another method. You could call it `morse_character_to_english`

Reply:
You can get the key from a value with `.key` or `.invert`